ai/ai21labs/v0/config/tasks.json

```diff
@@ -0,0 +1,1056 @@
+            "casual",
+            "formal",
+            "long",
+            "short"
+          ],
+          "example": "general",
+          "instillAcceptFormats": [
+            "string"
+          ],
+          "instillUIOrder": 2,
+          "instillUpstreamTypes": [
+            "value"
+          ],
+          "title": "Style",
+          "type": "string"
+        },
+        "start-index": {
+          "description": "The start index of the text to be paraphrased",
+          "instillAcceptFormats": [
+            "integer"
+          ],
+          "instillUIOrder": 3,
+          "instillUpstreamTypes": [
+            "value",
+            "reference"
+          ],
+          "title": "Start Index",
+          "type": "integer"
+        },
+        "end-index": {
+          "description": "The end index of the text to be paraphrased",
+          "instillAcceptFormats": [
+            "integer"
+          ],
+          "instillUIOrder": 4,
+          "instillUpstreamTypes": [
+            "value",
+            "reference"
+          ],
+          "title": "End Index",
+          "type": "integer"
+        }
+      },
+      "required": [
+        "text",
+        "style"
+      ],
+      "title": "Input",
+      "type": "object"
+    },
+    "output": {
+      "description": "Output",
+      "instillUIOrder": 0,
+      "properties": {
+        "suggestions": {
+          "description": "Model Output",
+          "instillUIOrder": 0,
+          "instillFormat": "array:string",
+          "instillUIMultiline": true,
+          "title": "Suggestions",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      },
+      "required": [
+        "suggestions"
+      ],
+      "title": "Output",
+      "type": "object"
+    }
+  },
+  "TASK_GRAMMAR_CHECK": {
+    "instillShortDescription": "Suggests changes to grammatical errors found by the model in the submitted string",
+    "description": "With AI21labs Grammatical Error Corrections (GEC) endpoint, you can tap into our advanced GEC engine, purpose-built and expertly tuned for this task.",
+    "input": {
+      "description": "Input",
+      "instillEditOnNodeFields": [
+        "text"
+      ],
+      "instillUIOrder": 0,
+      "properties": {
+        "text": {
+          "description": "The text content to be checked",
+          "instillAcceptFormats": [
+            "string"
+          ],
+          "instillUIMultiline": true,
+          "instillUIOrder": 1,
+          "instillUpstreamTypes": [
+            "value",
+            "reference",
+            "template"
+          ],
+          "title": "Text",
+          "type": "string"
+        }
+      },
+      "required": [
+        "text"
+      ],
+      "title": "Input",
+      "type": "object"
+    },
+    "output": {
+      "description": "Output",
+      "instillUIOrder": 0,
+      "properties": {
+        "suggestions": {
+          "description": "Model Output",
```

Comment:
nit
clearer?
"description": "Model Output",

The suggestions of the text to be corrected from model

Reply:
question
Same as above. Is there specific reason we do not follow the pattern defined by vendors?

![image](https://github.com/user-attachments/assets/d933a624-56ae-4dc2-a9b7-4973d9179872)

Reply:
Fixed, now all the tasks have structured output.